db/seeds.rb

```diff
@@ -1 +1,8 @@
+FactoryGirl.create_list(:student, 12)
+FactoryGirl.create_list(:coach, 3)
+FactoryGirl.create :mentor
+FactoryGirl.create :supervisor
```

Comment:
To keep things consistent with the other lines the usage of parenthesis here wouldn't be a bad thing.